orttraining/orttraining/core/framework/gradient_graph_builder.h

```diff
@@ -138,6 +138,8 @@ class GradientGraphBuilder {
   // Tracks tensors that are stashed in the forward pass for later use in backward pass.
   std::unordered_set<std::string> stashed_tensors_;
 
+  const std::unordered_set<size_t>* GetStopGradientEdges(const Node& node) const;
```

Comment:
nit, comment here 
return index for stop gradient input edges if found. 
return nullptr if no stop gradient edge is not found